.github/workflows/release.yaml

```diff
@@ -49,11 +49,10 @@ jobs:
         run: make release tag=${{ env.RELEASE_VERSION }}
 
       - name: Create Release
-        uses: lightninglabs/gh-actions/action-gh-release@2021.01.25.00
+        uses: lightninglabs/gh-actions/action-gh-release@2024.07.24.00
```

Comment:
nit: Why does the lnd commit refer to the gh-action by hash, and this commit by tag?

Reply:
Because Laolu merged before I was able to push the tag :sweat_smile: